plugins/k8smeta/src/plugin.cpp

```diff
@@ -428,11 +295,20 @@ bool my_plugin::init(falcosecurity::init_input& in)
     try
     {
         m_thread_table = t.get_table(THREAD_TABLE_NAME, st::SS_PLUGIN_ST_INT64);
+
+        // get the 'cgroups' field accessor from the thread table
+        m_thread_field_cgroups = m_thread_table.get_field(
```

Comment:
We need these to be able to correctly fetch cgroups string from the thread entry in the `capture_open()` listen CAP callback.